src/main/java/seedu/linkedout/logic/commands/ClearCommand.java

```diff
@@ -11,7 +11,7 @@
 public class ClearCommand extends Command {
 
     public static final String COMMAND_WORD = "clear";
-    public static final String MESSAGE_SUCCESS = "Address book has been cleared!";
+    public static final String MESSAGE_SUCCESS = "Job book has been cleared!";
```

Comment:
Small grammatical error, should change to better fit our product.

Reply:
a972860c86657043b96a8bc0fbd3c3823566ffc4